python/ray/train/_internal/storage.py

```diff
@@ -36,12 +36,25 @@
 
 logger = logging.getLogger(__file__)
 
+ARTIFACT_STORAGE_PATH_ENV_VARS = [
+    "RAY_ARTIFACT_STORAGE",
```

Comment:
How can users know where to set this? Would this be in the `RunConfig.storage_path` docstring?

Reply:
Hm, probably in the env vars section in the docs, or in the "Configuring persistent storage" part? Preferences?

Reply:
(I think in those two places would be good)

Reply:
Yeah that makes sense! Though we should link to the Configuring Persistent Storage page from the `storage_path` docstring, so that users can figure out what to expect when `storage_path=None`.